cli/azd/cmd/deploy.go

```diff
@@ -105,6 +105,7 @@ type deployAction struct {
 	middlewareRunner         middleware.MiddlewareContext
 	packageActionInitializer actions.ActionInitializer[*packageAction]
 	alphaFeatureManager      *alpha.FeatureManager
+	skipPackage              bool
```

Comment:
We shouldn't need this flag. Once we support the `--from-package` flag in my other PR @ https://github.com/Azure/azure-dev/pull/1833 then this would implicitly skip packaging.

Reply:
Removed